config/prod/phil-biaffect.yaml

```diff
@@ -0,0 +1,47 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2-v10.j2
+stack_name: iwanna-instance
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "SysBio"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "BiAffect"
+  # The resource owner
+  OwnerEmail: "phil.snyder@sagebase.org"
+
+  # Default account settings, leave alone unless you plan to override
+  VpcName: "computevpc"
+  KeyName: "scicomp"
+  AMIId: "ami-0de53d8956e8dcf80"      # Amazon Linux 2 AMI 2.0.20190313
+
+  # Settings have default values but can be overriden. Set the below
+  # parameters *only* if you want to override the defaults.
+
+  # (Optional) EC2 instance type, default is "t2.nano" (other available types https://aws.amazon.com/ec2/pricing/on-demand/)
+  # InstanceType: "t2.medium"
+  # (Optional) EC2 boot volume size in GB, default is 8GB, Max is 1000GB
+  # VolumeSize: "50"
```

Comment:
Are you ok with small disk size and type?

Reply:
yeah, we'll see how well it works. I'm just downloading and storing a few dozen table rows at a time.